packages/maskbook/src/utils/hooks/useImageFailover.ts

```diff
@@ -0,0 +1,28 @@
+import { useAsync } from 'react-use'
+
+interface ImgCdnPair {
+    img: string
+    cdn: string
+}
+
+export function useImageFailover(imgCdnPairs: ImgCdnPair[]) {
```

Comment:
```suggestion
export function useImageFailover(urls: string[]) {
```

Reply:
It needs the `img` to check whether `cdn` is reachable. src=ImgCdnPair.cdn would always error.